validator/constants.py

```diff
@@ -48,7 +48,7 @@
 
 DEFAULT_WEBAPP_MRKT_URLS = ["https://marketplace.mozilla.org",
                             "https://marketplace-dev.allizom.org"]
-BUGZILLA_BUG = "https://bugzilla.mozilla.org/show_bug.cgi?id=%d"
+BUGZILLA_BUG = "https://bugzil.la/%d"
```

Comment:
Because the full URLs are visible in the validation messages, and the longer URLs add a lot of noise and distracting line breaks.